runbotics/runbotics-common/src/model/api/process-credential.model.ts

```diff
@@ -12,3 +12,5 @@ export interface ProcessCredentialDto
     extends Omit<ProcessCredential, "credential"> {
     credential: CredentialDto;
 }
+
+export interface UpdateProcessCredential extends Pick<ProcessCredential, "id" | "order"> {};
```

Comment:
You're not adding anything, so there's no need to extend - you can:
export type UpdateProcessCredential = Pick\<ProcessCredential, "id" | "order"\>
instead - it will be more clear that you create type out of ProcessCredential, not declaring new one